### ISSUE TITLE
Empty `src` or `id` in figures.yaml breaks preview 

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire#634]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

In figures.yaml, remove the value for the `src` and/or `label` attribute. 

### Actual behavior

When either field is left blank, you will not be able to preview the project when running `quire preview`. The only way to prevent this error is to include `""` as the value.  

### Expected behavior

You should be able to leave these attributes blank without breaking the preview.

### Version numbers

Quire: 1.0.0-rc.0
Node: 18.12.1
nvm: 0.39.1
OS: Monterey 12.5.1

### What browsers are you seeing the problem on?

Chrome

### Relevant Terminal/Shell output

```shell
edunigan@GT31233 test-project % quire preview
[CLI] running eleventy using lib/11ty cli
[CLI:11ty] running eleventy serve
[CLI:11ty] projectRoot /Users/edunigan/test-project
[CLI:11ty] {
  config: '/Users/edunigan/test-project/.eleventy.js',
  input: 'content',
  output: '_site',
  epub: '_epub',
  data: '_computed',
  includes: '../_includes',
  layouts: '../_layouts',
  public: './public'
}
[quire] DEBUG	 Figures:ImageProcessor         
      inputRoot: /Users/edunigan/test-project/content/_assets/images
      outputRoot: /Users/edunigan/test-project/_site
    
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] The "path" argument must be of type string. Received null (via TypeError)
[11ty] 
[11ty] Original error stack trace: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
[11ty]     at new NodeError (node:internal/errors:393:5)
[11ty]     at validateString (node:internal/validators:163:11)
[11ty]     at Object.join (node:path:1172:7)
[11ty]     at new Figure (/Users/edunigan/test-project/_plugins/figures/figure/index.js:28:28)
[11ty]     at FigureFactory.create (/Users/edunigan/test-project/_plugins/figures/figure/factory.js:30:20)
[11ty]     at /Users/edunigan/test-project/_plugins/figures/index.js:19:28
[11ty]     at Array.map (<anonymous>)
[11ty]     at AsyncEventEmitter.<anonymous> (/Users/edunigan/test-project/_plugins/figures/index.js:18:50)
[11ty]     at /Users/edunigan/test-project/node_modules/@11ty/eleventy/src/Util/AsyncEventEmitter.js:19:61
[11ty]     at Array.map (<anonymous>)
[11ty] Wrote 0 files in 0.06 seconds (v2.0.0-canary.18)
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-mother-a.jpg
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/lange-mother-a.jpg'
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/lange-mother-a.jpg'
[quire] DEBUG	 Figures:ImageTiler             skipping previously tiled image '/Users/edunigan/test-project/content/_assets/images/figures/lange-mother-a.jpg'
[quire] INFO	 Figures:IIIF:Manifest          Generated manifest for figure "cat-1"
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/evans-fields-family-a.jpg
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/evans-fields-family-a.jpg'
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/evans-fields-family-a.jpg'
[quire] DEBUG	 Figures:ImageTiler             skipping previously tiled image '/Users/edunigan/test-project/content/_assets/images/figures/evans-fields-family-a.jpg'
[quire] INFO	 Figures:IIIF:Manifest          Generated manifest for figure "cat-2"
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-mother-b.jpg
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-mother-b.jpg
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-mother-c.jpg
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-mother-c.jpg
[quire] INFO	 Figures:IIIF:Manifest          Generated manifest for figure "mother-variants"
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-layers/original-label.png
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-layers/original-label.png
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-layers/additional-label.png
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-layers/additional-label.png
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-layers/scratches.png
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-layers/scratches.png
[quire] DEBUG	 Figures:Figure                 processing annotation image figures/lange-layers/thumb.png
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-layers/thumb.png
[quire] DEBUG	 Figures:ImageProcessor         processing inputPath: /Users/edunigan/test-project/content/_assets/images/figures/lange-layers/base.jpg
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/lange-layers/base.jpg'
[quire] DEBUG	 Figures:ImageTransformer       skipping previously transformed image '/Users/edunigan/test-project/content/_assets/images/figures/lange-layers/base.jpg'
[quire] DEBUG	 Figures:ImageTiler             skipping previously tiled image '/Users/edunigan/test-project/content/_assets/images/figures/lange-layers/base.jpg'
[quire] INFO	 Figures:IIIF:Manifest          Generated manifest for figure "mother-annotations"
```
```


### Supporting Information

_No response_

**Comment by Erin-Cecele** [comment]
In progress for v1.0.0-rc.1: https://jira.getty.edu/browse/DEV-13870

**Comment by Erin-Cecele** [comment]
Closed with PR #639
